src/group-configurations/group-configuration-container/utils.js

```diff
@@ -0,0 +1,61 @@
+import messages from './messages';
+
+/**
+ * Formats the given URL to a unit page URL.
+ * @param {string} url - The original part of URL.
+ * @param {string} courseId - The ID of the course.
+ * @returns {string} - The formatted unit page URL.
+ */
+const formatUrlToUnitPage = (url, courseId) => `/course/${courseId}${url}`;
+
+/**
+ * Retrieves a list of group count based on the number of items.
+ * @param {Array} items - The array of items to count.
+ * @param {function} formatMessage - The function for formatting localized messages.
+ * @returns {Array} - List of group count.
+ */
+const getGroupsCount = (items, formatMessage) => {
+  if (items?.length) {
+    const message = items.length === 1
+      ? formatMessage(messages.containsGroup, { len: items.length })
+      : formatMessage(messages.containsGroups, { len: items.length });
+
+    return [message];
+  }
+
+  return [];
+};
```

Comment:
```suggestion
/**
 * Retrieves a list of group count based on the number of items.
 * @param {Array} items - The array of items to count.
 * @param {function} formatMessage - The function for formatting localized messages.
 * @returns {Array} - List of group count.
 */
const getGroupsCountMessage = (items, formatMessage) => {
if (!items?.length) {
    return [];
  }

  const messageKey = items.length === 1 ? messages.containsGroup : messages.containsGroups;
  const message = formatMessage(messageKey, { len: items.length });

  return [message];
};
```

It should handle edge case at the beginning and name should reflect `message` for more clarity

Reply:
changed